tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
+from astropy import units as u
+import numpy as np
+
+import pytest
+
+import os
+
+data_path = os.path.join(tardis.__path__[0], 'io', 'tests', 'data')
+
+@pytest.fixture
+def artis_density_fname():
+    return os.path.join(data_path, 'artis_model.dat')
+@pytest.fixture
+def artis_abundances_fname():
+    return os.path.join(data_path, 'artis_abundances.dat')
+
+def test_simple_read_artis_density(artis_density_fname):
```

Comment:
Have a look how I use the fixtures here and use them in a similar manner in your tests (in the call-sign of the function)